src/classes/Database.php

```diff
@@ -65,33 +64,76 @@ public function connect(): \PDO {
         }
     }
 
-    private function createTables(): void {
-        Logger::debug("Creating database tables");
 
-        $sql = "CREATE TABLE IF NOT EXISTS users (
-            id INTEGER PRIMARY KEY AUTOINCREMENT,
-            username TEXT NOT NULL UNIQUE,
-            password TEXT NOT NULL,
-            created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
-        )";
+    public function getMigration(int $version): ?\Cronbeat\Migration {
```

Comment:
@jetbrains-junie move this to a migration helper class. rename to "loadMigration"